src/main/java/seedu/address/model/module/Semester.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.model.module;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Module's semester in the transcript.
+ * <p>
+ * Guarantees: immutable; is valid as declared in {@link #isValidSemester(String)}
+ */
+public class Semester {
```

Comment:
Similarly for this, please update the docs

Reply:
Yep, going to update documentations for both lucidchart and the developers guide once i'm done adding test.